### PR TITLE
add missing .

### PR DESCRIPTION
**Pull request by babangsund**
Oops. https://github.com/facebook/relay/pull/2846#issuecomment-533221917

### CODE REVIEW

**Comment by mrtnzlml** [comment]
OK, you did it faster, thanks. I'd personally add the test though since it's a fix of a fix. 😅

**Comment by sibelius** [comment]
can you add a test?

**Comment by babangsund** [comment]
Probably should. :-)

**Comment by babangsund** [comment]
> Probably should. - although uncertain exactly what you want to test?

Well, here's a tiny test for `createPrintRequireModuleDependency` 🤷‍♂

File: packages/relay-compiler/codegen/RelayFileWriter.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const graphql = require('graphql');
 const invariant = require('invariant');
 const nullthrows = require('nullthrows');
 const path = require('path');
-const writeRelayGeneratedFile = require('./writeRelayGeneratedFile');
+const {writeRelayGeneratedFile} = require('./writeRelayGeneratedFile');
```
Comment (sibelius) [inline packages/relay-compiler/codegen/RelayFileWriter.js]: why this?

Comment (babangsund) [reply packages/relay-compiler/codegen/RelayFileWriter.js]: So I could export the function we want to test 🤔

**Comment by sorenhoyer** [comment]
After this is merged, would it be possible to make a patch version and do an npm publish, so we don't have to build relay-compiler from master?

**Comment by mrtnzlml** [comment]
@sorenhoyer I think it still doesn't work anyway. We have to wait a bit more. I tried multiple times in the past but seems like refetch container still requires the last argument (which makes this unusable - at least for OSS).

**Comment by sibelius** [comment]
@mrtnzlml refetch on typescript is broken because of this https://github.com/facebook/relay/issues/2874

**Comment by mrtnzlml** [comment]
@sibelius Mm, so it won't work for custom compilers using ESM as well, right? (I use ESM generated output with Flow)

**Comment by sibelius** [comment]
after this https://github.com/facebook/relay/pull/2875 it will

File: packages/relay-compiler/codegen/RelayFileWriter.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ const graphql = require('graphql');
...
```
Comment (alunyov) [inline packages/relay-compiler/codegen/RelayFileWriter.js]: Yeah, this, unfortunately, won't work with our internal FB compiler setup. 

So we should probably keep default export from this `writeRelayGeneratedFile`.

File: packages/relay-compiler/codegen/__tests__/writeRelayGeneratedFile-test.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * @emails oncall+relay
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+
+const {
+  createPrintRequireModuleDependency,
```
Comment (alunyov) [inline packages/relay-compiler/codegen/__tests__/writeRelayGeneratedFile-test.js]: We should probably make `createPrintRequireModuleDependency` a standalone module, and import it to `writeRelayGeneratedFile`.

File: packages/relay-compiler/codegen/writeRelayGeneratedFile.js [hunk 4]
```diff
@@ -169,4 +169,7 @@ function extractHash(text: ?string): ?string {
   return match && match[1];
 }
 
-module.exports = writeRelayGeneratedFile;
+module.exports = {
+  createPrintRequireModuleDependency,
+  writeRelayGeneratedFile,
+};
```
Comment (alunyov) [inline packages/relay-compiler/codegen/writeRelayGeneratedFile.js]: Let's keep the old version of the export

File: packages/relay-compiler/codegen/__tests__/writeRelayGeneratedFile-test.js (hunk 3 above)
Comment (babangsund) [reply packages/relay-compiler/codegen/__tests__/writeRelayGeneratedFile-test.js]: This can fortunately be done in a jiffy. :-)

**Comment by babangsund** [comment]
> Thanks for adding the tests!

My pleasure.

I've moved `createPrintRequireModuleDependency` into a standalone module, renamed the test, and reverted the import/exports.

**Pull request closed without merging**